NebulaPatcher/Patches/Dynamic/StorageComponent_Patch.cs

```diff
@@ -61,7 +61,7 @@ public static void SetBans_Postfix(StorageComponent __instance, int _bans)
         [HarmonyPatch("Sort")]
         public static void Sort_Postfix(StorageComponent __instance)
         {
-            if (SimulatedWorld.Initialized && !StorageManager.EventFromServer && !StorageManager.EventFromClient)
+            if (SimulatedWorld.Initialized && !StorageManager.EventFromServer && !StorageManager.EventFromClient && GameMain.data.localPlanet != null)
```

Comment:
Could you please also add the null check to the functions above `AddItem`, `AddItemStacked`, `TakeItemFromGrid`?
Thanks a lot!

Reply:
Didn't do these initially because I didn't see a way they could be called without being on a plannet, but I've added them now.